crates/sncast/src/main.rs

```diff
@@ -379,7 +376,7 @@ fn update_cast_config(config: &mut CastConfig, cli: &Cli) {
 
     config.rpc_url = clone_or_else!(cli.rpc_url, config.rpc_url);
     config.account = clone_or_else!(cli.account, config.account);
-    config.keystore = clone_or_else!(cli.keystore, config.keystore);
+    config.keystore = cli.keystore.clone().or(config.keystore.clone());
```

Comment:
what's the difference between this and `clone_or_else!` macro?

Reply:
Clone_or_else is defined as 
```
$field.clone().unwrap_or_else(|| $config_field.clone())
```
So basically I use `or` instead of `unwrap_or_else`.